task/06-conditions-n-loops-tasks.js

```diff
@@ -296,7 +344,20 @@ function getDigitalRoot(num) {
  *   '{[(<{[]}>)]}' = true 
  */
 function isBracketsBalanced(str) {
-    throw new Error('Not implemented');
+    let arr = [];
+    let open = ['[', '(', '<', '{'];
+    let close = [']', ')', '>', '}'];
+
+    for (let i = 0; i < str.length; i++) {     
+        if (open.indexOf(str[i]) !== -1) {
+            arr.push(str[i]);
+            continue
+        }
+        
+        if (open.indexOf(arr.pop()) !== close.indexOf(str[i]))  return false;
```

Comment:
too long, move return false on the nesxt line, and add brackets

Reply:
Understand